app1.js

```diff
@@ -70,7 +70,7 @@ function userGuessingGame() {
     while (wannaPlay.toLowerCase() === YES_STRING) {
         let numberOfAttempts = 3;
 
-        for (let i = 1; i <= numberOfAttempts; i++) {
```

Comment:
This will match the final case of in addition to the previous ones and I don't think that's what you mean to do.  We want to give them 3 tries, not 4, right?